### ISSUE TITLE
Unable to run scenarios after upgrading to java 9

### ISSUE THREAD

**Issue by gkulikov** [prashant-ramcharan/courgette-jvm#86]
Trying to avoid classpath exceeding 32k symbols error I decided to update jdk to 9.0.4
When I execute following task:

>tasks.withType(Test) {
>    systemProperties = System.getProperties()
>    systemProperties.remove("java.endorsed.dirs")
>}

> task runScenariosInParallel(type: Test) {
>    useTestNG()
>
>    include '**/suite/TestSuite.class'
>    outputs.upToDateWhen { false }
>}

get following output:
> Task :runScenariosInParallel
>Error occurred during initialization of boot layer
>java.nio.file.InvalidPathException: Illegal char <:> at index 3: jrt:/modules
>Error occurred during initialization of boot layer
>java.nio.file.InvalidPathException: Illegal char <:> at index 3: jrt:/modules
>Gradle suite > Gradle test > suite.TestSuite.parallelRun FAILED
>    courgette.runtime.CourgetteTestFailureException
>
>There were failing tests. Refer to the Courgette html report for more details.
>courgette.runtime.CourgetteTestFailureException: There were failing tests. Refer to the Courgette html >report for more details.
>
>1 test completed, 1 failed
> Task :runScenariosInParallel FAILED`

Stacktrace:
[link](https://gist.github.com/gkulikov/6af7f13303e243a3c0080fbf06ec1811)

**Comment by prashant-ramcharan** [comment]
I think I may know what's causing this issue.... will provide a fix.

**Comment by prashant-ramcharan** [comment]
Can you retry with this version?

````gradle
repositories {
    jcenter()
}

testCompile 'io.github.prashant-ramcharan:courgette-jvm:2.4.5-snapshot'
````

**Comment by prashant-ramcharan** [comment]
Please try using Courgette version 3.0.0 which was just released.

**Comment by gkulikov** [comment]
Hi, @prashant-ramcharan ,

thanks for your help. Issue for java 9 is resolved (tried with 3.0.0).

I must say that I had no success with java 10 and 11, but that's a different story.

**Comment by prashant-ramcharan** [comment]
I tested it with Java 11 and it works fine.

Will close this issue now.
